weather_dl/weather-dl

```diff
@@ -25,7 +25,14 @@ import tempfile
 import weather_dl
 
 if __name__ == '__main__':
-    logging.getLogger().setLevel(logging.INFO)
+    logger = logging.getLogger()
+    fmt = '%(levelname)s %(asctime)s %(name)s: %(message)s'
+    datefmt = '%y-%m-%d %H:%M:%S'
+    formatter = logging.Formatter(fmt=fmt, datefmt=datefmt)
+    handler = logging.StreamHandler()
+    handler.setFormatter(formatter)
+    logger.root.addHandler(handler)
+    logger.setLevel(logging.INFO)
```

Comment:
This should probably be moved to the `configure_logger` function. 

https://github.com/google/weather-tools/blob/ecb194c7bc62d9d0ad3982cce24eb745a470be84/weather_dl/download_pipeline/pipeline.py#L48